assets/svelte/components/PageAstNode.svelte

```diff
@@ -115,8 +115,18 @@
       {#each node.content as subnode, index}
         <svelte:self node={subnode} nodeId="{nodeId}.{index}" />
       {/each}
+      {#if isDragTarget && $draggedObject}
+        <div class="dragged-element-placeholder p-2">{@html $draggedObject.body}</div>
```

Comment:
Can we just use tailwind here?

```suggestion
        <div class="outline-dotted outline-2 outline-red-500 p-2">{@html $draggedObject.body}</div>
```

Reply:
Nope. Because this is part of the web component that uses the stylesheet compiled from the used classes in the template. And since that class is not used anywhere then it's not available.

Reply:
Gotcha and is `p-2` available?

Reply:
It happened to be but on a second thought I don't what that artificial padding